src/portal/lib/nodes.ts

```diff
@@ -418,13 +387,27 @@ export async function getDNodes(api: any, address: string) {
         lat: node.location.latitude,
       },
       resources: {
-        cru: node.resourcesTotal.cru,
-        mru: node.resourcesTotal.mru,
-        hru: node.resourcesTotal.hru,
-        sru: node.resourcesTotal.sru,
+        cru: node.total_resources.cru,
+        mru: node.total_resources.mru,
+        hru: node.total_resources.hru,
+        sru: node.total_resources.sru,
+      },
+      usedResources: {
+        cru: node.used_resources.cru,
+        mru: node.used_resources.mru,
+        hru: node.used_resources.hru,
+        sru: node.used_resources.sru,
       },
       pubIps: ips,
+      rentContractId: node.rentContractId,
+      rentedByTwinId: node.rentedByTwinId,
+      rentStatus: await getRentStatus(node.nodeId, currentTwinID)
```

Comment:
this method will do a call for each node we have to know its status, and we already can get this from the node object

Reply:
removed the function and stored the condition in rentStatus variable.